src/group-constructors.js

```diff
@@ -55,6 +55,8 @@
     var underlyingPlayer;
     var ticker = function(tf) {
       var player = underlyingPlayer._wrapper;
+      if (player.playState == 'pending') return;
```

Comment:
In https://github.com/web-animations/web-animations-next/blob/master/src/effect-callback.js#L34 we probably want to also call the effect callback when the playState changes.